### PR TITLE
Tomo script enhance 1

### PR DESCRIPTION
**Pull request by torsten-systemticks**

### CODE REVIEW

**Comment by torsten-systemticks** [comment]
I would write a test, but I do not find the infrastructure for testing scripts (i.e. xtend code)

**Comment by torsten-systemticks** [comment]
> 
> 
> Maybe we should add a simple unit test, also for documentation purposes.

Test now available. Please check latest commit

**Comment by torsten-systemticks** [comment]
> What semantics does the '?' have?

a?.b

means, do only execute a.b if a != null (syntacic sugar feature from xtend)

**Comment by torsten-systemticks** [comment]
> I am not sure if it is a problem with committing the files or if the formatting is also off locally. I think we should fix that before merging.

I cannot find a formatting problem...

File: src/com.elektrobit.ebrace.core.scriptimporter/files/RaceScripts/src/api/Examples.xtend [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import api.ScriptBase
+import com.elektrobit.ebsolys.script.external.ScriptContext
+import com.elektrobit.ebsolys.script.external.Execute
+import com.elektrobit.ebsolys.script.external.Execute.ExecutionContext
+
+class Examples
+{
+
+	extension ScriptContext _scriptContext
+    extension ScriptBase _scriptBase
```
Comment (sebastian-systemticks) [inline src/com.elektrobit.ebrace.core.scriptimporter/files/RaceScripts/src/api/Examples.xtend]: In the GitHub view, it looks like indentions are different between lines 11 and 12. Maybe it is due to tabs and spaces being used.

File: src/com.elektrobit.ebrace.core.scriptimporter/files/RaceScripts/src/api/ScriptBase.xtend [hunk 2]
```diff
@@ -222,6 +223,58 @@ class ScriptBase {
...
+	{
+		val p = path.replaceAll('\\[\\s*(\\d+)\\s*\\]', '\\.\\<$1\\>')
+		
+		if(path.startsWith('$.'))
+		{
+			event.decode.decodedTree.rootNode.extract(p.substring(2))			
+		}
+		else
+		{
+			event.decode.decodedTree.rootNode.extract(p)
```
[the remark this answers is not in the document]
Comment (torsten-systemticks) [reply src/com.elektrobit.ebrace.core.scriptimporter/files/RaceScripts/src/api/ScriptBase.xtend]: I see. I am going to check that against my local changes.
It was a copy past from built-in EB solys editor into Eclipse IDE editor

[the remark this answers is not in the document]
Comment (torsten-systemticks) [reply src/com.elektrobit.ebrace.core.scriptimporter/files/RaceScripts/src/api/ScriptBase.xtend]: Strange. I don't have the formatting issues in my local environment

**Pull request merged**